docs/user/check-ref/kubernetes_wrapper.rst

```diff
@@ -0,0 +1,243 @@
+        :rtype: list
+
+.. py:function:: statefulsets(name=None, replicas=None, **kwargs)
+
+        ::
+
+        Return list of Statefulsets.
+
+        :param name: Statefulset name.
+        :type name: str
+
+        :param replicas: Statefulset replicas.
+        :type replicas: int
+
+        :param **kwargs: Statefulset :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of Statefulsets. Typical Statefulset has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: replicasets(name=None, replicas=None, **kwargs)
+
+        ::
+
+        Return list of ReplicaSets.
+
+        :param name: ReplicaSet name.
+        :type name: str
+
+        :param replicas: ReplicaSet replicas.
+        :type replicas: int
+
+        :param **kwargs: ReplicaSet :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of ReplicaSets. Typical ReplicaSet has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: deployments(name=None, replicas=None, ready=None, **kwargs)
+
+        ::
+
+        Return list of Deployments.
+
+        :param name: Deployment name.
+        :type name: str
+
+        :param replicas: Deployment replicas.
+        :type replicas: int
+
+        :param ready: Deployment ready status.
+        :type ready: bool
+
+        :param **kwargs: Deployment :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of Deployments. Typical Deployment has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: configmaps(name=None, **kwargs)
+
+        ::
+
+        Return list of ConfigMaps.
+
+        :param name: ConfigMap name.
+        :type name: str
+
+        :param **kwargs: ConfigMap :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of ConfigMaps. Typical ConfigMap has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: persistentvolumeclaims(name=None, phase=None, **kwargs)
+
+        ::
+
+        Return list of PersistentVolumeClaims.
+
+        :param name: PersistentVolumeClaim name.
+        :type name: str
+
+        :param phase: Volume phase.
+        :type phase: str
+
+        :param **kwargs: PersistentVolumeClaim :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of PersistentVolumeClaims. Typical PersistentVolumeClaim has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: persistentvolumes(name=None, phase=None, **kwargs)
+
+        ::
+
+        Return list of PersistentVolumes.
+
+        :param name: PersistentVolume name.
+        :type name: str
+
+        :param phase: Volume phase.
+        :type phase: str
+
+        :param **kwargs: PersistentVolume :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of PersistentVolumes. Typical PersistentVolume has "metadata", "status" and "spec".
```

Comment:
It is definitely returned by the API

```python
In [1]: pv.obj.keys()
Out[1]: dict_keys(['spec', 'status', 'metadata'])
```

``$ kubectl get pv pvc-1cca9cf3-... -o yaml``

```yaml
...
status:
  phase: Bound
```